### PR TITLE
[Havoc DH] level 99 talents

### PR DESCRIPTION
**Pull request by yajinni**
Added the 3 talents and suggestions for them.

image
image

image
image

image
image

### CODE REVIEW

File: src/parser/demonhunter/havoc/modules/talents/BlindFury.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    this.active = this.selectedCombatant.hasTalent(SPELLS.BLIND_FURY_TALENT.id);
+    if (!this.active) {
+      return;
+    }
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.EYE_BEAM), this.onEyeBeamsCast);
+  }
+
+  //Since this will always max out your fury, we only care how much did you gain.
+  onEyeBeamsCast(event) {
+    this.gained += (MAX_FURY - (event.classResources[0].amount - event.classResources[0].cost));
```
Comment (HawkCorrigan) [inline src/parser/demonhunter/havoc/modules/talents/BlindFury.js]: Would it be useful to count the wasted still?
Casting EB while being nearly capped is still bad right?

Comment (yajinni) [reply src/parser/demonhunter/havoc/modules/talents/BlindFury.js]: Reached out to the guide writer on discord to see what he would consider a good cut off.

Comment (HawkCorrigan) [reply src/parser/demonhunter/havoc/modules/talents/BlindFury.js]: Do you want to merge it now anyway or wait on feedback?

Comment (yajinni) [reply src/parser/demonhunter/havoc/modules/talents/BlindFury.js]: Luckily he replied quickly and said the cut off is 50 fury. So will be adding that in.

**Pull request merged**